### ISSUE TITLE
Support Docker secrets

### ISSUE THREAD

**Issue by Yajo** [caprover/caprover#921]
**Is your feature request related to a problem? Please describe.**
Yes, I want to expose some secrets to only 1 user inside the container.

**Describe the solution you'd like**
Caprover should support [swarm secrets](https://docs.docker.com/engine/swarm/secrets/). These are also supported in [Docker Compose v3 spec](https://docs.docker.com/compose/compose-file/#secrets).

**Describe alternatives you've considered**
- Using env variables. This exposes the secret to all users in the container.
- Using a separate bootstrapping container that writes the secret and permissions to a file in a shared volume.
- Skip caprover and use an alternate packaging format, such as `docker stack deploy` or https://github.com/docker/app.

**Comment by githubsaturn** [comment]
Please read [Goals and Scope](https://github.com/caprover/caprover/blob/master/CONTRIBUTING.md#important-caprover-goals--scope) of the project

TLDR:
CapRover does not aim to expose all Docker capabilities. Having said that, if you need such capabilities, CapRover allows you to customize the behavior. For secrets, you can either use Portrainer or Docker CLI as a one click app to create secrets and attack them to your service. Once attached, CapRover does not modify it.

**Comment by omars44** [comment]
Docs PR: https://github.com/caprover/caprover-website/pull/125

Apparently it is supported via https://caprover.com/docs/service-update-override.html
